tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_basic_models.py

```diff
@@ -10,31 +10,19 @@ def test_dfn_well_posed(self):
         model = pybamm.lithium_ion.BasicDFN()
         model.check_well_posedness()
 
-        copy = model.new_copy()
```

Comment:
What was the point of these tests, originally?

Reply:
just coverage